flank-bash/scripts/updateLibs/updateLlvm.main.kts

```diff
@@ -0,0 +1,77 @@
+@file:CompilerOptions("jvmTarget = '1.8'")
+
+@file:Import("../util/downloadFiles.main.kts")
+@file:Import("../util/archive.main.kts")
+@file:Import("../util/OsHelper.kts")
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardCopyOption
+
+private val currentPath = Paths.get("")
+private val llvmPath = Paths.get(currentPath.toString(), "llvm")
+
+fun updateLlvmWindows() {
+    val llvmExe = Paths.get(llvmPath.toString(), "LLVM-win64.exe")
+    if (llvmExe.toFile().exists()) {
+        println("LLVM exists")
+    } else {
+        println("Downloading Windows LLVM...")
+        llvmPath.toFile().mkdirs()
+        downloadFile(
+            url = "https://releases.llvm.org/8.0.0/LLVM-8.0.0-win64.exe",
+            destinationPath = llvmExe.toString()
+        )
+    }
+
+    llvmExe.toFile().extract(llvmPath.toFile(), "zip", "xz")
```

Comment:
We need to make a ticket for this. As far as i am aware this wont extract correctly

Reply:
it needs work as its complicated and needed this task to be comleted first

Reply:
@Sloox I think that you just test this for #1134 ticket, if yes we should remove this code and make task for it.

Reply:
I created TODO in code